README.md

```diff
@@ -53,6 +53,12 @@ You will also find detailed [guides](./docs/Guides.md) and [frequently asked que
 
 ### Prerequisite: Bitcoin Core
 
+We chose to rely as much as possible on bitcoin core, to interface with and monitor the blockchain and also to manage onchain funds (eclair does not include an onchain wallet, channel opening transactions are funded by your bitcoin core node, and channel closing transactions return funds to your bitcoin core node).
```

Comment:
nit: we're capitalizing `Bitcoin Core` everywhere else, and use `on-chain` instead of `onchain` in documentation